### ISSUE TITLE
Mobile Phone Placeholder

### ISSUE THREAD

**Issue by Sydney-o9** [chronotruck/FlagPhoneNumber#109]
At the moment the placeholder shows a landline phone number, is it possible to use a mobile phone number placeholder?

**Comment by grifas** [comment]
`FlagPhoneNumber` uses `libPhoneNumber` lib and what I know about it now, it's not possible to use a mobile phone number.

**Comment by Sydney-o9** [comment]
Yes you can @grifas - see [NBPhoneNumberUtil](https://github.com/iziz/libPhoneNumber-iOS/blob/master/libPhoneNumber/NBPhoneNumberUtil.m#L2095).

So you could do the following in your [FPNTextField](https://github.com/chronotruck/FlagPhoneNumber/blob/master/Sources/FPNTextField.swift#L411)
```
let example = try phoneUtil.getExampleNumber(forType: countryCode.rawValue, type: NBEPhoneNumberType.MOBILE)
```
